examples/edge-functions/supabase/functions/cloudflare-turnstile/index.ts

```diff
@@ -0,0 +1,61 @@
+// Follow this setup guide to integrate the Deno language server with your editor:
+// https://deno.land/manual/getting_started/setup_your_environment
+// This enables autocomplete, go to definition, etc.
+
+import { serve } from 'https://deno.land/std@0.131.0/http/server.ts'
```

Comment:
any reason to use `std@0.131.0` instead of the latest?

Reply:
This is what the `supabase functions new` template sets.